packages/core/test/rendererCreateWithTheme.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```

Comment:
This is just moving it out of the global test setup file so it can be typed. We should probably make it a separate task to rewrite all the tests to use RTL.